src/internal/connector/onedrive/collection.go

```diff
@@ -370,6 +372,7 @@ func (oc *Collection) populateItems(ctx context.Context, errs *fault.Bus) {
 
 					// check for errors following retries
 					if err != nil {
+						logger.Ctx(ctx).With("error", err.Error()).Infow("unable to download item")
```

Comment:
Nit: avoid statements like "unable to".  This should be an error log, which will already make explicit that we were unable to / failed to do the thing.
```suggestion
						logger.Ctx(ctx).With("error", err.Error()).Error("downloading item")
```